server/jamovi/server/instance.py

```diff
@@ -746,6 +751,9 @@ def _on_dataset_ins_cols(self, request, response):
             column.transform = column_pb.transform
             column.parent_id = column_pb.parentId
 
+            if column.column_type == ColumnType.DATA:
+                self._data.set_cells_as_edited(column, 0, self._data.row_count - 1)
```

Comment:
possible for this to get pushed inside column? or not?

Reply:
My initial response is no because there are instances where type NONE is needed as well but other places where it is not.